test/script/reference.t

```diff
@@ -54,6 +54,8 @@
   24
   24
   13
+  owi: [ERROR] expected multiple memories but got (unknown module )
+  [54]
```

Comment:
This is a little bit annoying because it prevents the remaining tests to run. I believe we should either:

1. update the submodule to a latest version where it may have been fixed
2. stop running this tests if the assumption is that an engine supporting the multi-memory proposal should only run the multi-memory tests and not the old ones (does the multi-memory proposal directory includes a copy of the old ones without the tests failing on multiple memories?)

Have you tried 1. ?

Reply:
I agree, the issue with updating the submodule is that its structure significantly changed, so simply updating the repository leads to many tests (all the multi-memory ones for example as well others) becoming "missing" files.
But I will take a look, maybe I can find an intermediary commit that fixes the issues and for which the structure of the files has not completely changed

Reply:
But there are indeed copies of the tests under the `multiple memories` folder were the only differences is that the tests that fail when there are multiple memories have been removed, so it seems that solution 2 is the more straightforward one.
I will do a quick cleanup of duplicate tests for now, we can open a separate issue for the updating of the repository since it is more work.

Reply:
Yes, it will be better to handle this separately.

> But there are indeed copies of the tests under the multiple memories folder were the only differences is that the tests that fail when there are multiple memories have been removed, so it seems that solution 2 is the more straightforward one.

I think this is the best solution. :)